server/models/commenthistory.js

```diff
@@ -12,6 +12,7 @@ export default (sequelize, DataTypes) => {
   CommentHistory.associate = (models) => {
     CommentHistory.belongsTo(models.Comment, {
       foreignKey: 'commentId',
+      useJunctionTable: false,
```

Comment:
@TheoOkafor why are we using `useJunctionTable: false` here? I thought this was for the one to many relationship between comment and comment?

Reply:
@PriscillaSam It is set to `false`. I am not sure about the Sequelize default value for the `useJunctionTable`. I had to state the required value explicitly.